README.md

```diff
@@ -20,14 +20,18 @@ A place to collate all the community made content for Monster Hunter World: The
 - Adds feystone drops which can be appraised into decorations
 - Adds a slot system to allow you to equip decorations to use on quests
 
+## Mod Development
```

Comment:
Can't quite believe I'm being this picky, but can we go sentence case? Ie no capital D here

Reply:
And should it be `Mods - under development`

Reply:
Haha sure, I think I've made the capitalization change now? And it's both mods that are still in development, as well as tools for helping people to develop mods, so I think my title is more appropriate. Couldn't find a way to fit that into an existing category.